README.md

```diff
@@ -8,42 +8,66 @@ Unlike offline alignment methods, for which state-of-the-art implementations are
   
 We aim to provide efficient reference implementations of score followers for use in real-time applications which can be easily integrated into existing projects.
 
+The full documentation for matchmaker is available online at [readthedocs.org](https://pymatchmaker.readthedocs.io/).
+
+
 ## Setup
 
+### Prerequisites
+
+- Available Python version: 3.9, 3.10, 3.11, 3.12 (3.12 recommended)
+- [Fluidsynth](https://www.fluidsynth.org/)
+- [PortAudio](http://www.portaudio.com/)
+
+Please ensure that you've installed the above packages before proceeding.
+You should not install `fluidsynth` using `pip install fluidsynth` as it is not compatible with `matchmaker`.
```

Comment:
Maybe we can change this sentence to 

```
First, install Fluidsynth, and then install the `pyfluidsynth` Python library. Note that `pyfluidsynth` only provides Python bindings for Fluidsynth; it does not install Fluidsynth itself. Be aware that there is also a `fluidsynth` Python library (without the `py-` prefix), but it is not compatible with `matchmaker`.
```

Reply:
(this refers to the sentence `You should not install fluidsynth...`)

Reply:
Thanks! Suggestions are updated here: https://github.com/pymatchmaker/matchmaker/pull/13/commits/7ac5682d9676603a376419e633587e7386907c71